communication/slack-guidelines.md

```diff
@@ -1,90 +1,213 @@
-- A link to the posting or job description
-- The business name that will employ the Kubernetes hire
-- The location of the role or if remote is OK
+## Moderation
 
-## DM (Direct Message) Conversations
+### Admin Expectations and Guidelines
 
-Please do not engage in proprietary company specific conversations in the Kubernetes Slack instance. This is meant for conversations around related Kubernetes open source topics and community. Proprietary conversations should occur in your company Slack and/or communication platforms.  As with all communication, please be mindful of appropriateness, professionalism, and applicability to the Kubernetes community.
+Admins should adhere to the general Kubernetes project [moderation guidelines].
 
+Admins should additionally make sure to mention they are a Slack admin and their
+timezone in their “What I do” section of their Slack profile.
 
-Note:
-We archive the entire workgroup's slack data in zip files when we have time. [The latest archive is from June 2016-November 2018.](https://drive.google.com/drive/folders/1idJkWcDuSfs8nFUm-1BgvzZxCqPMpDCb?usp=sharing)
+Be mindful of how you handle communication during stressful interactions.
+Administrators act as direct representatives of the community, and need to
+maintain a very high level of professionalism at all times. If you feel too
+involved in the situation to maintain impartiality or professionalism, that’s
+a great time to enlist the help of another admin.
+
+Try to take any situations that involve upset or angry members to DM or video
+chat. Please document these interactions for other Slack admins to review.
+
+Content will be automatically removed if it violates code of conduct or is a
+sales pitch. Admins will take a screenshot of such behavior in order to document
+the situation. The community takes such violations extremely seriously, and the
+admins are empowered to handle it swiftly.
+
+
+### Sending Messages to the Channel
+
+`@here` and `@channel` should be used rarely. Members will receive notifications
+from these commands. Remember Kubernetes is a global project - please be kind.
+Note: `@all` is restricted and can only be used by admins.
+
+
+### Reviewing Slack Requests
+
+Admins are tasked with reviewing requests for channels and other things such as
+bots, tokens or webhook requests.
+
+
+#### Reviewing Channel Requests
+
+Channel requests should be reviewed for their relation and relevance to the
+Kubernetes community. Typically channels should be dedicated to SIGs, WGs, UGs,
+sub-projects, community topics, and other things related to Kubernetes programs
+and projects.
+
+Small external projects are encouraged to use the channel of the SIG, WG, or UG
+most relevant to them. Other things such as programming language specific
+channels are discouraged and should in turn be steered to `#kubernetes-client`
+or communication avenues commonly used by their specific language.
+
+In general, use your best judgment.
+
```

Comment:
can we add a line that admins should pin the code of conduct to the channel at the time of creation? 

example:  
We abide by the CNCF Code of Conduct. Contact conduct@kubernetes.io if we can help. Join #slack-admins for other matters. Enjoy.

Reply:
Added a note about that.